### DISCUSSION TITLE
machine vs human user with IAM_OWNER role

### DISCUSSION THREAD

**Discussion by musicformellons** [zitadel/zitadel#8217] [Q&A] [answered]
In the related `steps.yaml` example I've found I see comments on Zitadel creating either machine or human user with IAM_OWNER role. I have a self-bootable instance of Zitadel working with a machine user. If I would like in addition to have a human user to do settings via the GUI, I suppose I let the machine user create a human user with IAM_OWNER role and I'm ready to go..., is that correct? Or would I run into issues maybe?

**Comment by hifabienne** [comment, marked as answer]
Hi @musicformellons 
We just recently introduced the possibility to setup a new ZITADEL instance with both a human and a service user.
But yes, otherwise this would be correct to setup with one user, and then use that to create others.

**Reply by fforootd** [reply]
I think we added this in 2.53.0

**Reply by musicformellons** [reply]
AFAICT this needs updating then: 
https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L581
https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L602

**Reply by fforootd** [reply]
Right, I created this PR for that https://github.com/zitadel/zitadel/pull/8229

**Comment by musicformellons** [comment]
Sounds good, will try again then!

**Reply by musicformellons** [reply]
Yep, you can bootstrap both at once now.
